### DISCUSSION TITLE
Task with frame

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4109] [Q&A]
while creating a task with more than 1 image(example 3 images), we get start_frame as 0 in the engine_segment table of the database. How can we make the start_frame as 1 as default while posting the image?

**Comment by nmanovic** [comment]
@keshva23 , I need more details to answer on the question. Why do you need that?

**Reply by keshva23** [reply]
Thanks @nmanovic for the reply.  I have clear query in the name of keshavadk above this query with the title as "Frame starting from 0 in task page". can you please refer that?
